modules/cdk-app/lib/cdk-demo-stack.ts

```diff
@@ -0,0 +1,15 @@
+import cdk = require('@aws-cdk/core');
+import s3 = require('@aws-cdk/aws-s3');
+
+export class DemoCdkStack extends cdk.Stack {
+  constructor(scope: cdk.Construct, id: string) {
+    super(scope, id);
+
+    new s3.Bucket(this, 'reInventDemoBucket', {
```

Comment:
Should the bucket name be configurable so tests running in parallel (e.g., two jobs in CircleCi) don't have name clashes?

Reply:
The static string here is a prefix identifier for any resources created by this template. The actual bucket name would be provided in the properties, and since I haven't included it, the name is randomized by default. But good observation nonetheless.